CONTRIBUTING.md

```diff
@@ -6,6 +6,10 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+For technical issues related to using Node.js that are not issues with the
```

Comment:
Yes, go ahead with a second commit on top of this one. (You or I or someone else can squash 'em all down into one commit later.)

Thanks for tackling this, by the way!